index.html

```diff
@@ -0,0 +1,138 @@
+            <img class="vect" src="./assets/Vector.png" alt=""/>
+           <div class="projects">
+               <div class="proj proj1">
+                <img src="./assets/stage.ico" alt="lecture"/>
+                <h3>Curtain Raise</h3>
+                <p>Perfomance by upcoming artists to bring in the music mood</p>
+               </div>
+               <div class="proj proj2">
+                <img src="./assets/lecture.ico" alt="lecture"/>
+                <h3>Sponsors Speech</h3>
+                <p>Sponsors and partners speech to everyone before competition</p>
+             </div>
+             <div class="proj proj3">
+                <img src="./assets/handshake.ico" alt="lecture"/>
+                <h3>Battle / competition</h3>
+                <p>Every band is given the stadge to showcase their music and expertise</p>
+             </div>
+              <div class="proj proj4">
+                <img src="./assets/medal.ico" alt="lecture"/>
+                <h3>Judges Choose the winner</h3>
+                <p>Judges choose the best band according to their showcase expertise.</p>
+            </div>
+           </div>
+           <button class="btn-proj">JOIN THE CONCERT</button>
+           <h5 class="see-all">SEE THE WHOLE PROGRAM</h5>
+        </section>
+        <section id="performance">
+            <div class="performers">
+                <h3>Featured Performers</h3>
+                <img src="./assets/Vector.png" alt=""/>
+                <div class="contestants">
+                   
+                       <!--modal start-->
+                       <div class="modal">
+                            <div class="modal-content">
+                                 <div class="modal-header">
+                                    <h2 class="band-name">Sauti sol band</h2>
+                                      <span class="close-modal">&times;</span>                                     
+                                 </div>
+                                 <div class="modal-body">
+                                     <div class="modal-image-holder">
+                                      <img class="band-image" src="./assets/sautisol.jpg" alt="sautosl band"/>
+                                    </div>
+                                    <div class="modal-info">
+                                        <h6 class="descr">Swahili soul band to sing in kenya</h6>
+                                        <img src="./assets/Vector.png" alt=""/>
+                                      <h6 class="paragraph">They sing sweet soulmusic in kiswahili very soothing to be presice.</h6>
+                                    </div>
+                                 </div>
+                            </div>
+                       </div>
+                       <!--modal stop-->
+                </div>
+            </div>
+            <button class="read-more">MORE<i class="fa fa-angle-down"></i></button>
```

Comment:
[OPTIONAL] Kindly make the speakers when the more button pressed appear above it not below it for a better UI experience
![image](https://user-images.githubusercontent.com/24703171/163507839-9cf473e9-f43a-41de-80da-be1dc57a4503.png)